specification/ai/Azure.AI.Client/agents/main.tsp

```diff
@@ -0,0 +1,2 @@
+import "./models.tsp";
+import "./routes.tsp";
```

Comment:
No need for this... we should have only one "main.tsp" in the root folder of project, that imports all that is needed.

Reply:
It follows what we have in OpenAI Assistants. If we restructure it, the comparisons will be harder in future.

Reply:
restructured